vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1022,15 +1069,80 @@ public void run() {
         mSearchView.setIconifiedByDefault(false);
         mSearchView.setOnQueryTextListener(this);
 
-        if (null != mFloatingActionButton) {
-            mFloatingActionButton.setOnClickListener(new View.OnClickListener() {
+//        if (null != mFloatingActionButton) {
+//            mFloatingActionButton.setOnClickListener(new View.OnClickListener() {
+//                @Override
+//                public void onClick(View v) {
+//                    Fragment fragment = getSelectedFragment();
+//
+//                    if (!(fragment instanceof AbsHomeFragment) || !((AbsHomeFragment) fragment).onFabClick()) {
+//                        onFloatingButtonClick();
+//                    }
+//                }
+//            });
+//        }
+
+        Drawable drawable;
+
+        drawable = getResources().getDrawable(R.drawable.ic_person_black_24dp).mutate();
+        DrawableCompat.setTint(drawable, getResources().getColor(android.R.color.white));
+        mFabStartChat.setIconDrawable(drawable);
+
+        drawable = getResources().getDrawable(R.drawable.ic_add_white).mutate();
+        DrawableCompat.setTint(drawable, getResources().getColor(android.R.color.white));
+        mFabCreateRoom.setIconDrawable(drawable);
+
+        drawable = getResources().getDrawable(R.drawable.riot_tab_rooms).mutate();
+        DrawableCompat.setTint(drawable, getResources().getColor(android.R.color.white));
+        mFabJoinRoom.setIconDrawable(drawable);
+
+        // Pre-Lollipop does not support elevation so cannot have touch guard
+        // above bottomNavigationView.
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            mFloatingActionsMenu.setOnFloatingActionsMenuUpdateListener(new FloatingActionsMenu.OnFloatingActionsMenuUpdateListener() {
                 @Override
-                public void onClick(View v) {
-                    Fragment fragment = getSelectedFragment();
+                public void onMenuExpanded() {
+                    touchGuard.setAlpha(0.6f);
 
-                    if (!(fragment instanceof AbsHomeFragment) || !((AbsHomeFragment) fragment).onFabClick()) {
-                        onFloatingButtonClick();
-                    }
+                    touchGuard.setOnClickListener(new View.OnClickListener() {
+                        @Override
+                        public void onClick(View v) {
+                            mFloatingActionsMenu.collapse();
+                        }
+                    });
+                }
+
+                @Override
+                public void onMenuCollapsed() {
+                    touchGuard.setAlpha(0);
+
+                    touchGuard.setOnClickListener(null);
```

Comment:
This will not remove the interception of click event, so the user will be stuck :)
You also have to call `setClickable(false)`

Reply:
Done

Reply:
I think this is still bugged. You have to call setClickable() AFTER setOnClickListener(). Look in setOnClickListener implementation